Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.5
+// swift-tools-version:5.7
```

Comment:
funny that this even worked

Reply:
Can you elaborate? Why wouldn't it?

Reply:
We updated SwiftPM to 5.7 a while ago, so I think the expected behavior would have been that SourceKit-LSP started failing back then.